spec/requests/api/v2/app_groups_spec.rb

```diff
@@ -219,4 +219,60 @@
       expect(json_response['affected_app']).to eq(2)
     end
   end
+  describe 'Delete Helm Infrastructure' do
+    it 'should delete the Helm Infrastructure' do
+      # Create a Helm Infrastructure
+      helm_infrastructure = create(:helm_infrastructure, status: HelmInfrastructure.statuses[:active])
+
+      # Send a DELETE request to the delete action
+      delete api_v2_helm_infrastructure_delete_path(id: helm_infrastructure.cluster_name), headers: headers
```

Comment:
since we move to app_group, it should be `api_v2_app_group_<<patch|post>>_path` and also please use `cluster_name:helm_infrastructure.cluster_name` instead of `id`